src/core/Button/Button.tsx

```diff
@@ -57,9 +57,13 @@ const StyledButton = styled(
   }: ButtonProps & { right?: boolean }) => (
     <CompButton
       {...passProps}
-      className={classnames(className, {
-        [`${baseClassName}--${variant}`]: variant !== 'default',
-      })}
+      className={classnames(
+        className,
+        {
+          [`${baseClassName}--${variant}`]: variant !== 'default',
+        },
+        fullWidth ? 'fi-button--fullWidth' : '',
```

Comment:
Uses classnames-function here: https://github.com/JedWatson/classnames
It takes format {'class-name': condition} and can be included using new key in the same object as variant is implemented

Reply:
Thanks. Fixed this one as well.